### PR TITLE
Add script to check for missing .gitattributes rules

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,22 @@ Consider including them if they are applicable to your project.
 
 You can use [this handy dandy generator](https://richienb.github.io/gitattributes-generator) to generate your gitattributes files on the fly based on files inside of this repository.
 
+## `.gitattributes` CI check
```
Comment (Richienb) [inline README.md]:
```suggestion
## CI step
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,22 @@ Consider including them if they are applicable to your project.
...
+
+If you want to check all files have explicit git attributes set, you may use the following script:
```
Comment (Richienb) [inline README.md]:
```suggestion
To check if all files have a corresponding rule in .gitattributes, this script can be used:
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,22 @@ Consider including them if they are applicable to your project.
...
+
+```sh
+missing_attributes=$(git ls-files | git check-attr -a --stdin | grep "text: auto")
+if [[ "$missing_attributes" ]]; then
+  echo ".gitattributes entry missing for the following files:";
+  echo "$files";
+else
+  echo "All files have git attributes.";
```
Comment (Richienb) [inline README.md]:
```suggestion
  echo "All files have a corresponding rule in .gitattributes";
```

File: README.md [hunk 4]
```diff
@@ -8,6 +8,22 @@ Consider including them if they are applicable to your project.
...
 You can use [this handy dandy generator](https://richienb.github.io/gitattributes-generator) to generate your gitattributes files on the fly based on files inside of this repository.
 
+## `.gitattributes` CI check
+
+If you want to check all files have explicit git attributes set, you may use the following script:
+
+```sh
+missing_attributes=$(git ls-files | git check-attr -a --stdin | grep "text: auto")
+if [[ "$missing_attributes" ]]; then
+  echo ".gitattributes entry missing for the following files:";
```
Comment (Richienb) [inline README.md]:
```suggestion
  echo ".gitattributes rule missing for the following files:";
```

File: README.md [hunk 5]
```diff
@@ -8,6 +8,22 @@ Consider including them if they are applicable to your project.
...
+  echo "$files";
```
Comment (Richienb) [inline README.md]: Where is `$files` defined?

**Review by Richienb — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (torkleyy) [reply README.md]: Ooops, fixed now.

**Pull request merged**